e2e/failureDetails-property/myreporter.js

```diff
@@ -0,0 +1,8 @@
+class MyCustomReporter {
+  onRunComplete(contexts, results) {
+    console.log('Custom reporter output:');
+    console.log(results.testResults[0].testResults.map(f => f.failureDetails));
```

Comment:
```suggestion
    console.log(results.testResults[0].testResults.map(f => JSON.stringify(f.failureDetails, null, 2)));
```

should make sure it's printed properly, no?

Reply:
Yeah, or using `utils.expect`.